docs/WritingTests.asciidoc

```diff
@@ -90,18 +246,19 @@ sub run {
     assert_screen "desktop", 300;
 }
 
-sub test_flags {
+sub test_flags() {
     return {fatal => 1};
 }
 
 1;
 -------------------------------------------------------------------
 
-=== Test Case Examples
-[id="testcase_examples"]
+==== Install software via `zypper`
+[id="testmodule_perl_zypper"]
```

Comment:
I don't know why you changed that and, sorry, that does not make sense. Keep in mind that openQA is a generic product so not specific to SLE/openSUSE hence "zypper" is merely an example here, not something that is important to explain in the context of openQA